_episodes_rmd/06_day5.Rmd

```diff
@@ -0,0 +1,266 @@
+- Capstone lesson
+    1. Revisit your spatial question
+    2. Load libraries and import data
+    3. Crop raster with a vector
+    4. Summarize raster data by vector locations
+- Keypoints and final questions (10-15 minutes)
+
+
+# 1) Revisit your spatial question
+
+On day 1 we brainstormed questions you could ask with geospatial data. You thought about questions you can answer with raster data, and separately, questions you could answer with vector data. Through this workshop you have practiced working with spatial data and have the tools to answer your question.
+
+In this final day we will implement tools you learned for combining raster and vector data and the types of questions you can answer when doing so.
+
+A spatial question may look like the following:
+
++ where does something occur?
++ how does something change across space or time?
++ what are the differences between locations?
+
+Examples of spatial questions are:
+
++ What county has the highest recorded precipitation?
++ Has mean annual temperature increased in the state?
++ How does the precipitation vary between Milwaukee and Madison?
+
+### Capstone data question: Does it get hotter in Milwaukee or Madison?
+
+In this capstone lesson we will use the following capstone data:
+
++ World Climate raster of maximum temperature (will download)
++ Natural Earth populated places vector point data for cities in Wisconsin (provided `ne_10m_populated_places_Wisconsin.csv` file)
+
+We will use the vector data to crop the raster data to the extent of Wisconsin and summarize the raster data using the point locations of Madison and Milwaukee.
+
+**If you are using your own data**, please have raster and vector data that cover a similar geographic area. Since we are cropping the raster data by the vector data, it will work best if your vector data is of a smaller geographic area than the raster data.
+
+# 2) Load libraries and import data
+For raster data, we use functions in the `raster` package. For vector data, you will use functions in the `sf` package. In addition, because we will visualize our data, we will use the plotting package `ggplot2`.
+
+> ## Solution
+> ```{r load package, message=FALSE}
+> 
+> library(raster)
+> library(sf)
+> library(ggplot2)
+> ```
+{: .solution}
+
+
+## Load raster data
+The capstone data includes the World Climate Data or "WorldClim". We will use your coding skills to access the worldclim dataset from the `raster` package, crop it to our extent of interest, and summarize the data. By the end of this section you should have your raster data as a 'Formal class RasterLayer' in your Global Environment
+
+> ## World Climate Data
+>
+> You can learn about and download the WorldClim from the website: https://www.worldclim.org/data/worldclim21.html 
+> For an example of how to use the `raster` package to access the WorldClim (raster) data and GADM (vector) data, visit this tutorial: https://www.gis-blog.com/r-raster-data-acquisition/ 
+> 
+>
+> {: .source}
```

Comment:
```suggestion
```

I don't think you need the `{: .source}` tag here if you don't have source code in the callout.

Reply:
@sstevens2 can you confirm?

Reply:
oh yeah you could be right. I'll leave this comment here for now.